cmds/wifi/wifi.go

```diff
@@ -38,34 +38,59 @@ func init() {
 	}
 }
 
+func generateConfig(a []string) (conf []byte, err error) {
```

Comment:
The convention I'm used to is a...string but I guess this is ok?

Reply:
Fixed